### PR TITLE
Development

### PR DESCRIPTION
**Pull request by musaeed**
Fixed readme and granted authorities

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,11 +192,11 @@ where curlJson.txt contains:
 ### Cook
 
  * Get one specific record
- ``GET http://localhost:8080/api/secure/cooks/{id}``
+ ``GET http://localhost:8080/api/public/cooks/{id}``
```
Comment (yar-malik) [inline README.md:195]: mate, above in Readme.md cooks is secure and here public.

File: README.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ Following is a list of some examples of rest requests one can use:
 ```
 GET: http://localhost:8080/api/secure/cooks
```
Comment (yar-malik) [inline README.md]: secure?

File: src/main/resources/query.sql [hunk 3]
```diff
@@ -3,6 +3,7 @@ CREATE TABLE login
   id serial NOT NULL,
   username character varying NOT NULL,
   password character varying NOT NULL,
+  granted_role varying NOT NULL,
```
Comment (yar-malik) [inline src/main/resources/query.sql]: why not character varying. The format is static. 
Consistency is better, I believe

**Comment by yar-malik** [comment]
Mate, also in readme there are many places where secure api has to be changed into public.

**Review by yar-malik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (musaeed) [reply README.md:195]: fixed

**Pull request merged**